infra/cifuzz.py

```diff
@@ -40,49 +51,114 @@ def main():
   build_fuzzer_parser = subparsers.add_parser(
       'build_fuzzers', help='Build an OSS-Fuzz projects fuzzers.')
   build_fuzzer_parser.add_argument('project_name')
-  build_fuzzer_parser.add_argument('repo_name')
+  build_fuzzer_parser.add_argument('github_repo_name')
```

Comment:
I think these need to be begun with `--` otherwise they are positional.

Reply:
Outdated, not using argparse anymore.